src/browsers/schema/content/homepage.md

```diff
@@ -0,0 +1,9 @@
+Examples - Gene: [SETD1A](link), Variant: [16-30992057-CAG-C](link)
+
+The Schizophrenia Exome Sequencing Meta-analysis (SCHEMA) consortium is a large multi-site collaboration dedicated to aggregating, generating, and analyzing high-throughput sequencing data of schizophrenia patients to improve our understanding of disease architecture and advance gene discovery.  
+
+The Phase I data set (GRCh37/hg19) contains the analysis of exomes from 24,248 cases and 97,322 controls, and _de novo_ mutations from 3,402 parent-proband trios. These results have pinpointed genome-wide significant results associating ultra-rare protein-coding variants in individual genes to risk of schizophrenia.  
+
+We would also like to thank the many tens of thousands of patients and families who generously contributed to our effort. This project is made possible by the generosity of many funders, including the [Stanley Foundation](https://www.broadinstitute.org/stanley), and [National Institutes of Mental Health (NIMH)](https://www.nimh.nih.gov/index.shtml), and the leadership of its members. The contributing groups to this effort are listed [here](about.md), and the terms of use are seen [here](terms.md).
+
+Exome results browser and analysis data last updated September 10th, 2020.
```

Comment:
Should this line be the last time anything about the browser was updated? Or the last time the data was updated (August 31st)?

Reply:
We can keep them separate.

Reply:
Actually let's just keep the browser instead.